node/src/rpc.rs

```diff
@@ -56,12 +56,7 @@ where
 	C::Api: frame_rpc_system::AccountNonceApi<Block, AccountId, Nonce>,
 	C::Api: pallet_transaction_payment_rpc::TransactionPaymentRuntimeApi<Block, Balance>,
 	C::Api: BlockBuilder<Block>,
-	C::Api: pallet_dex_rpc::DexRuntimeApi<
-		trappist_runtime::opaque::Block,
-		trappist_runtime::AssetId,
-		trappist_runtime::Balance,
-		trappist_runtime::AssetBalance,
-	>,
+	C::Api: pallet_dex_rpc::DexRuntimeApi<Block, AssetId, Balance, Balance>,
```

Comment:
just a comment, in the future we should remove this one and access the dex runtime api with state_call.